docroot/modules/custom/webform_serialization/src/Normalizer/WebformNormalizer.php

```diff
@@ -36,4 +38,20 @@ public function getFields($entity, $bundle, ResourceType $resource_type) {
     return $enabled_public_fields;
   }
 
+  /**
+   * Add the options for select items if the options are stored in config files.
+   *
+   * @param array $formItems
```

Comment:
Although these technically are form items, let's just call this variable `$elements`, since we're dealing with webforms.